src/index.ts

```diff
@@ -93,6 +94,9 @@ async function main() {
             run.status !== 'completed' &&
             new Date(run.created_at) < cancelBefore,
         );
+        if (cancel_only_queued) {
+          runningWorkflows.filter(run => run.status === 'queued');
+        }
```

Comment:
Looking at this implementation, I wonder if it would be better to add an option like:

```yaml
cancel_status: 'queued'
```

Then you could change line 94 to:

```js
cancel_status ? run.status ===  cancel_status  : run.status !== 'completed'
```

That might provide a more flexible API when someone wants to cancel a different status in the future.

Reply:
Made the changes, works as expected!